containers/Overview/index.tsx

```diff
@@ -124,15 +124,17 @@ class Overview extends React.Component<Props> {
           </Box>
         </Box>
         <Box basis={'1/2'} margin={{ left: 'large' }}>
-          <Box>
-            <Heading level="4" margin={{ top: 'small', bottom: 'small' }}>Asset Originator Details</Heading>
-
-            <Text margin={{ top: 'xsmall' }}> {name} </Text>
-            <Text margin={{ top: 'medium' }}> The following information is provided by the Asset Originator: </Text>
+          <div>
+            <Heading level="4" margin={{ top: 'small', bottom: 'small' }}>The following information is provided by the Asset Originator</Heading>
```

Comment:
Please change this back to:
<h4>Asset Originator Details</h4>
<p>The following information is provided by the Asset Originator:</p>